software/rfcs/rocketFind.md

```diff
@@ -0,0 +1,62 @@
+# RocketFind
+
+RocketFind is our system for finding rockets once they've landed.
+
+## Pre-reading
+
+We've tried several GPS based approaches to finding rockets in the past. 
+
+In 2021, we had a custom avionics solution which would send us a text message with the GPS coordinates. Unfortunately we weren't able to launch APEX I then, and due to the temperamental nature of the avionics (mostly the SIM800L board), we decided to replace it with an off the shelf solution.
+
+In 2022, we used a Vodafone Curve tracker to find the rocket. The Curve would allow us to access the rocket's GPS coordinates via an app. The Curve service has a small monthly fee. We encountered some issues with this system when we tried to find APEX I. 
+
+1. We didn't receive any coordinates for 40ish minutes, most likely due to the patch network conditions at SARA. 
```

Comment:
We had solid readings while waiting on the launchpad - my personal theory is that it shit itself duting the actual launch and had to restart and took a while to get another lock. There are GPS modules which are rated to not shut down when going supersonic/very high, highly doubt the curve had one of these but we could buy one to use ourselves

Reply:
Even when it landed, weren't the readings a while apart?